index.rst

```diff
@@ -240,6 +240,101 @@ fitSrcMatchAstrometry
+   Initial versions of everything - astrometry, photometry, probably PSFs.
+   Probably aperture corrections of some kind, but targeted specifically at making compensated apertures work for FGCM.
+   These are all attached to its image output, ``initial_pvi``, which is a lot like today's ``calexp``.
+   This will be a fluence image with nJy pixel units, but how much of that is done by this task vs. ISR is TBD.
+   Its output catalog, ``initial_source_detector``, will be converted to Parquet before it is written, but not SDM-standardized.
+   Footprints will be written to a separate ``SourceCatalog`` dataset, ``initial_footprints``.
+   Whether this task does multiple detection rounds (to iterate on CR detection, the source detection filter, or the detection threshold) is TBD; we'd like to minimize that.
+   We would like to avoid running a deblender here (if we do, it will have to happen after we have the PSF model), but this depends on how this performs on crowded fields.
+   We are not thrilled with the name of this task and would love ideas for improvements (which should be coordinated with the names for ``bootstrapVisit``, ``finalizeImage``, ``compressImage``, and ``rebuildImage``).
+
+bootstrapVisit
+   Consolidate the per-detector outputs of ``bootstrapImage`` and recover from failures on some detectors by using those that succeeded (especially for WCSs).
+
+associateIsolatedStars
+   Pretty much just a renamed ``IsolatedStarAssociationTask``.
+
+solveAstrometry
+   Pretty much just a generic label for ``jointcal`` and ``GBDES``.
+
+fgcm
+   Nothing new here, except the sharding and names of the output datasets.
+
+modelVisitBackground
+   A replacement for SkyCorrectionTask, probably using PCA.
+   SkyCorrectionTask will serve as a placeholder until we have something better.
+   Also takes care of subtracting the wings of bright stars.
+
+finalizeCharacterization
+   Nothing new here except new names for the output datasets.
+
+   We are using "characterizations" here to mean "PSFs and aperture corrections", and are not thrilled about that, but we need something that means "PSFs and aperture corrections" that could also absorb aperture corrections being done rather differently than they are today.
+
+finalizeAstrometry
+   This is a re-run of the astrometry solver with slightly different connections; I think it'll be necessary to make our (achromatic) WCSs consistent with the chromatic PSF models that we'll someday produce in ``finalizeCharacterization``.
+   But this needs more thought.
+
+updateVisitSummary
+   Nothing new here (though the current task is only a few weeks old, and apparently still has some bugs).
+
+finalizeImage
+   This new task takes all of our hard-won final characterizations and calibrations of the image and produces a final ``{visit, detector}`` ``pvi`` image and full-depth, all-measurements ``final_source_detector`` catalog.
+   The latter will be SDM-standardized before it is ever written to disk.
```

Comment:
I think it is always dangerous to embed final in any term, because you risk the dissertation problem of ``finalizeImageV2Updated``

Reply:
But these aren't files or datasets; they're dataset types, and the idea here is that those represent interfaces, not particular task outputs.  So you don't add "V2Updated", you reconfigure the original task that produced `finalizedImage` to write `penultimateImage` (or whatever) instead and let the new final task start writing `finalizedImage`.